### DISCUSSION TITLE
Live Mosher - front-end for ffglitch-core

### DISCUSSION THREAD

**Discussion by pawelzwronek** [ramiropolla/ffglitch-core#6] [Show and tell]
Hi @ramiropolla !
I've just published the alpha version of my [app](https://github.com/pawelzwronek/LiveMosher).
I would be grateful if you could test it and share any thoughts about it. Right now, it has a basic input transcoding configuration, but I think I should add more options. I'm looking forward to your reply.

**Comment by ramiropolla** [comment]
Hi @pawelzwronek. I'm looking forward to testing it!

But I'm kind of on vacation at the moment :). I'll have a look in a couple of weeks.

**Reply by pawelzwronek** [reply]
Sure, no rush. Have a good time!

**Comment by bgola** [comment]
hi @pawelzwronek great work ! I'm jumping on the conversation because I also started working on a similar idea focused on live coding / changing scripts on the fly. you might want to check it out: 

https://github.com/bgola/ffglitch-livecoding

**Comment by pawelzwronek** [comment]
@bgola cool idea, especially for an audio artists 🤘 I was thinking abut adding some knobs or sliders to my app, and connect then with a variables in the script, and you have already did this with a nice tricky way 👌

**Comment by pawelzwronek** [comment]
@bgola One more question: have you tested Live Mosher by any chance? Were there any issues running it?

**Comment by pawelzwronek** [comment]
@ramiropolla I was wandering if you consider adding Live Mosher to the list of frontends on https://ffglitch.org, I would be grateful.

**Reply by ramiropolla** [reply]
I definitely will! Sorry I haven't had time to look more into it yet.

**Reply by pawelzwronek** [reply]
Thanks, no rush!
